arangod/Agency/State.cpp

```diff
@@ -207,17 +204,16 @@ bool State::persistConf(index_t index, term_t term, uint64_t millis,
     THROW_ARANGO_EXCEPTION(res);
   }
 
-  OperationResult logResult, confResult;
   try {
-    logResult = trx.insert("log", log.slice(), _options);
-    confResult = trx.replace("configuration", configuration.slice(), _options);
+    OperationResult logResult = trx.insert("log", log.slice(), _options);
```

Comment:
There isn't (and never was) any check for the value of `logResult`. So it may go wrong and this will not be noticed.

Reply:
I guess we should bail out in that case as well.